Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,29 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where is that? 
```

Comment:
What is `that`? Could this be rephrased?

Reply:
`that` is `Current`, the subject of the previous line, which I prefer to avoid to repeat.

Reply:
```suggestion
### Where can I find it?
```

Reply:
Also this section feels more like an overview that just a where to find the file. Maybe something like 

```suggestion
### Overview
```

Reply:
I accepted this but I don't agree using first or second person. 
I've checked and I don't see any of the Cookbook using this way to explain. (expect for the Olivier's about framework).